docs/navigation.md

```diff
@@ -3,72 +3,26 @@ id: navigation
 title: Navigating Between Screens
 ---
 
-Mobile apps are rarely made up of a single screen. Managing the presentation of, and transition between, multiple screens is typically handled by what is known as a navigator.
+By default, React Native renders your components onto a single screen. To create multiple screens and navigate between them you need a navigation library. The following table helps you choose the one that's right for you. Each column identifies a different feature and each row represents a different navigation library (React Router Native isn't included because it doesn't have stack navigation). These features aren't the be-all and end-all of what makes a good navigation library but they are important. Keep reading to find out why.
```

Comment:
Could we either include React Router Native and add a column for "stack natigation", or list all the the features that are uses as criteria used for inclusion. For example:

> Navigation libraries included below must all implement x, y, and z. These features aren't the be-all and end-all of what makes a good navigation library but they are important to cover a broad range of user experiance expectations.

Reply:
I think we shouldn't include React Router Native. It's not really feature-complete enough to use for production apps, imo.